pkg/restmapper/restmapper_test.go

```diff
@@ -60,11 +60,13 @@ func TestPartialRestMapperProvider(t *testing.T) {
 
 		// Create two different REST mappers with different passed group filter predicates
 		allGroupsRestMapperProvider := NewPartialRestMapperProvider(AllGroups)
-		allGroupsRestMapper, err := allGroupsRestMapperProvider(restCfg)
+		httpClient, err := rest.HTTPClientFor(restCfg)
+		g.Expect(err).To(gmg.BeNil())
```

Comment:
Idiomatically this would be
```suggestion
		g.Expect(err).ToNot(gmg.HaveOccurred())
```